PULL_REQUEST.md

```
@@ -0,0 +1,108 @@
+docker run --hostname=ff67fd77cb8d --env=PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin --env=REDISBLOOM_ARGS= --env=REDISEARCH_ARGS= --env=REDISJSON_ARGS= --env=REDISTIMESERIES_ARGS= --env=REDIS_ARGS= --network=bridge -p 6379:6379 -p 8001:8001 --restart=no --label='desktop.docker.io/wsl-distro=Ubuntu' --label='org.opencontainers.image.ref.name=ubuntu' --label='org.opencontainers.image.version=22.04' --runtime=runc -d redis/redis-stack:latest
+```
+
+Por padrão, o banco ficará como o host `localhost` e a porta `6379`, e somente isso é necessário para poder rodar ele com o caching local. Caso deseje utilizar um banco de dados Redis em produção, será necessário alterar as variáveis de ambiente `REDIS_PRODUCTION_HOST`, `REDIS_PRODUCTION_PORT`, `REDIS_PRODUCTION_USER` e `REDIS_PRODUCTION_PASSWORD` no arquivo `.env` com os dados fornecidos pela Redis Cloud ou outro provedor do banco.
+
+O Container do Redis vem junto com o Redis-Insight, que permite visualizar os dados armazenados no cache da aplicação. Para acessá-lo, basta abrir o navegador e ir até a URL `http://localhost:8001`.
+
+A aplicação depende de pelo menos um banco de dados MySQL para ser rodada. Para isso, basta utilizar o Shell do MySQL ou um cliente de banco de dados para criar o banco de dados `corenotes_db`, e inserir os dados de autenticação. Após criar o banco, copie a sua string de conexão na variável `DATABASE_URL`.
+Caso queira rodar os testes unitários, crie um segundo banco de dados `corenotes_db_test` e copie a sua string de conexão na variável `SHADOW_DATABASE_URL` para o Prisma poder reconhecê-lo na hora de rodar os testes dos endpoints da API.
+
+Por fim, é necessário preencher algumas variáveis de ambiente para o Nest e os providers utilizarem. Utilize o valor "development" ou "local" para a chave `NODE_ENV`, o valor "mysql" para a chave `DATABASE_PROVIDER`, uma porta de quatro dígitos para a chave `PORT`, a url da API e do front-end nas chaves `API_URL` e `FRONTEND_URL` e a chave para a criação de JWTs na variável `APP_SECRET`. Por segurança, pode-se usar uma das chaves geradas no link acima.
+
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)